packages/providers/mgt-electron-provider/src/Authenticator/SimpleCachePlugin.ts

```diff
@@ -0,0 +1,64 @@
+// NOTE : This is a simple cache plugin made for the purpose of demonstrating caching support for the Electron Provider.
```

Comment:
Run `yarn setLicense` at root of repo to apply the license header to all new source files